### DISCUSSION TITLE
Aqara temperature, humidity and pressure sensor cannot rejoin network

### DISCUSSION THREAD

**Discussion by aram535** [Koenkk/zigbee2mqtt#15754] [Devices]
I replaced the battery in a Aqara Temp/Hum sensors (aka Xiaomi WSDCGQ11LM) and it refused to send data through.  So I tried removing it, then tried force removing it.  Which worked... but now it is refusing to re-join.

* Long Reset the device
* Short Reset the device
* Moved the sensor next to the controller to make sure of a good link. 
* Removed and reinstalled the battery on the sensor.
* Restarted Z2M (edge).
* Checked Z2M configuration -- it is not banned (no banned devices)
* Replaced battery with a new battery (orientation verified)

There are 7 other of the same sensors on the network with no issues and I have replaced batteries in others.  I'm at a loss on how to debug this.

**Comment by yangli8866** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by muc2810** [comment]
@aram535 

Was ist possible for you to solve the problem. I have exactly the same issue.

**Reply by aram535** [reply]
I ended up sending that unit back to Amazon and using another one.

**Reply by lux73** [reply]
@muc2810 

> Was ist possible for you to solve the problem. I have exactly the same issue.

Try remove Battery & wait some Hours before reattach then re-pair Sensor

**Comment by muc2810** [comment]
I already removed the battery over night, but nothing worked. I‘ll send them back to amazon and try new ones. ThanksStefan Von meinem iPhone gesendetAm 18.07.2024 um 21:59 schrieb Michael ***@***.***>:﻿

@aram535
Was ist possible for you to solve the problem. I have exactly the same issue.

Try remove Battery & wait some Hours before reattach then re-pair Sensor
